File: shopcar/src/main/java/com/cxyz/car/ui/activity/mainfragment/MainActivity.java
package com.cxyz.car.ui.activity.mainfragment;

import android.os.Bundle;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.cxyz.car.R;

public class MainActivity extends AppCompatActivity {

    private Button btn1_afragment;
    private MainFragment mainFragment;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.fragment_main);

        mainFragment=new MainFragment();
        getFragmentManager().beginTransaction().add(R.id.fl_fragment,mainFragment).commitAllowingStateLoss();
    }
}

File: main/src/main/java/com/cxyz/main/ui/activity/MainActivity.kt
package com.cxyz.main.ui.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.alibaba.android.arouter.facade.annotation.Route
import com.cxyz.main.R

@Route(path = "/main/MainActivity")
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity_main_layout)
    }

}